src/_data/toc/installation-guide.yml

```diff
@@ -83,10 +76,16 @@ pages:
         - label: Install the Magento software
           url: /install-gde/install/cli/install-cli-install.html
 
-        - label: Contributing developers—update, reinstall Magento
+        - label: (Contributor) Update or reinstall Magento
```

Comment:
This section of the TOC has a lot of tasks now. Maybe we should consider changing the title of this topic to something more generic, like "Contributor workflows"? That might not be the best title, but it's all I can come up with at the moment.

Reply:
I know you can only do so much at one time, but consider moving update/reinstall info to the Software Update Guide.